app/Services/PluginImportService.php

```
@@ -0,0 +1,598 @@
+                    'name' => $settings['name'] ?? 'Imported Plugin',
+                    'trmnlp_id' => $trmnlpId,
+                    'data_stale_minutes' => $settings['refresh_interval'] ?? 15,
+                    'data_strategy' => $settings['strategy'] ?? 'static',
+                    'polling_url' => $settings['polling_url'] ?? null,
+                    'polling_verb' => $settings['polling_verb'] ?? 'get',
+                    'polling_header' => isset($settings['polling_headers'])
+                        ? str_replace('=', ':', $settings['polling_headers'])
+                        : null,
+                    'polling_body' => $settings['polling_body'] ?? null,
+                    'markup_language' => $markupLanguage,
+                    'render_markup' => $fullLiquid,
```

PHPStan (2.1.33)

Variable $fullLiquid might not be defined.


(variable.undefined)